qa-tests-backend/src/test/groovy/AdmissionControllerTest.groovy

```diff
@@ -99,8 +106,17 @@ class AdmissionControllerTest extends BaseSpecification {
         // Wait for propagation to sensor
         sleep(10000 * (ClusterService.isOpenShift4() ? 4 : 1))
 
-        // Pre run scan to avoid timeouts with inline scans in the tests below
-        ImageService.scanImage(NGINX_IMAGE)
+        // Pre run scan to avoid registry timeouts with inline scans in the
+        // tests below.
+        ImageService.scanImage(SCAN_INLINE_IMAGE_NAME_WITH_SHA)
+
+        // Wait until we received metadata the inline scan image. This will
+        // ensure that enrichment has finalized.
+        withRetry(18, 10) {
```

Comment:
This is 180s, i.e. 3 minutes right? How about making this 15 minutes? Or is `scanImage` going to give up sooner anyway?

Reply:
`scanImage()` defaults to 150s. globalTimeout is 2000s. I'm ok to change these values (150s + 180s) but would prefer to do that if these calls show up as flakes.